relay/relay.go

```diff
@@ -163,8 +167,9 @@ func (rs *RelayService) HandleMsg(packet *message.RelayPacket) error {
 	if nextHop == nil {
 		clients := websocket.GetServer().GetClientsById(destID)
 		if clients == nil {
-			// TODO: handle client not exists
-			return errors.New("Client Not Exists: " + hex.EncodeToString(destID))
+			log.Info("Client Not Online:", hex.EncodeToString(destID))
+			rs.addRelayPacketToBuffer(destID, packet)
```

Comment:
Any space limitation for  buffer the packets

Reply:
Not yet. Currently it's unlimited.